Firmware/ultralcd.cpp

```diff
@@ -1692,22 +1700,27 @@ static void pgmtext_with_colon(const char *ipgmLabel, char *dst, uint8_t dstSize
     dst[dstSize-1] = '\0';      // terminate the string properly
 }
 
+//! @brief Show Extruder Info
+//!
+//! @code{.unparsed}
+//! |01234567890123456789|
+//! |Nozzle FAN: 0000 RPM|	FAN c=10 r=1  SPEED c=3 r=1
+//! |Print FAN:  0000 RPM|	FAN c=10 r=1  SPEED c=3 r=1
+//! |Fil. Xd:000 Yd:000  |	Fil. c=4 r=1
+//! |Int:  000 Shut: 000 |	Int: c=4 r=1  Shut: c=4 r=1
+//! ----------------------
+//! @endcode
+//! @todo Positioning of the messages and values on LCD aren't fixed to their exact place. This causes issues with translations.
```

Comment:
doxygen @todo line isn't needed anymore as it was fixed with https://github.com/prusa3d/Prusa-Firmware/pull/2202
Or am I wrong?

Reply:
> @DRracer I think everything looks correct. I am not sure if all of the doxygen @todo comments have been fixed by #2202 . Can you have a look again?

@3d-gussner We'll perform tests in the upcoming days, if the dialogs are now OK in every language, the TODO's may be removed.

> I will try to document as much as i can find.
> Really happy that community changes/PRs got more attention now and it is fun to do that again more frequently.

Thank you, we're happy to have such a great community.

> It would be great if we could add to the existing `.../lang/` scripts or an new script that checks the actual code if `////c=xx r=yy` is missing in new code generated, so the owner of the code gets notified.

The lang-check.py is the most recent script to check correct sizes of translated strings in the code. It still has problems with multiline texts though...

> Even better it would be to move all output messages to `messages.c/h` as @leptun does with his #2092

Please wait with further refactoring for the next major release (3.9). There are numerous other PR's to merge.

> The `translations.md` is a good start but can be improved over time.
> Also the `lang-import.sh` may need some attention as I just added to me known languages special characters replacements.

I agree